swerve/TorqueSwerveX.java

```diff
@@ -33,7 +33,7 @@ public final class TorqueSwerveX extends TorqueSwerveModule {
             NEOFreeSpeedRPS = 5676 * 0.9 / 60,
             driveWheelFreeSpeedRps = (NEOFreeSpeedRPS * wheelDiameter * Math.PI) / driveGearRatio,
             // driveMaxSpeed = 4.6, turnPGain = 1.5, turnIGain = 0.5, turnDGain = 0,
-            driveMaxSpeed = 4.6, turnPGain = 1.25, turnIGain = 0, turnDGain = 0,
+            driveMaxSpeed = 4.6, turnPGain = 1.5, turnIGain = 0, turnDGain = 0,
```

Comment:
No way these are tuned well

Also this needs to be a config like last years swerve. You cant just hardcode these things into the swerve--different robots have different properties.

Reply:
You need to make a config class like last year so that a user of the module can override things themself

Reply:
You tuned these with davey... and i can make a config if it makes you feel better but they're all going to use the same default config like last year anyways

Reply:
> No way these are tuned well

Yea I just thought yall tuned them more then that buts its ok cos we will retune anyways

👍 besides that all good